tsconfig.json

```diff
@@ -36,7 +37,7 @@
       "@radix-ui/react-progress": ["packages/react/progress/src"],
       "@radix-ui/react-radio-group": ["./packages/react/radio-group/src"],
       "@radix-ui/react-roving-focus": ["./packages/react/roving-focus/src"],
-      "@radix-ui/react-scroll-area": ["./packages/react/react-scroll-area/src"],
+      "@radix-ui/react-scroll-area": ["./packages/react/scroll-area/src"],
```

Comment:
same here, it was probably always wrong, discovered it when I included scroll area in the ssr tests

Reply:
These problems are making me wonder more and more why we need these if they weren't highlighted by anything